__init__.py

```diff
@@ -11,11 +11,12 @@
     "support": 'TESTING',
     "category": "Import-Export"}
 
+import importlib
 
 if "bpy" in locals():
-    import importlib
     importlib.reload(blendergltf)
-
+if 'filters' in globals():
```

Comment:
There is no need to check for filters in the globals. Just leave the import where it was and move the reload of filters under the bpy branch so it looks like the reload for the blendergltf module.